src/Tolerance/Bridge/Guzzle/Operation/RetryEvaluator/PromiseRetryEvaluator.php

```diff
@@ -0,0 +1,30 @@
+<?php
+
+/*
+ * This file is part of the Tolerance package.
+ *
+ * (c) Samuel ROZE <samuel.roze@gmail.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Tolerance\Bridge\Guzzle\Operation\RetryEvaluator;
+
+use GuzzleHttp\Psr7\Response;
+use Tolerance\Operation\RetryEvaluator\RetryEvaluator;
+
+class PromiseRetryEvaluator implements RetryEvaluator
+{
+    /**
+     * {@inheritdoc}
```

Comment:
Any chance to allow injecting the statusCode as array that should lead to a retry ?

On our side, we would retry only 502 response for example.